app/src/controllers/log/constants.js

```diff
@@ -42,6 +42,11 @@ export const LAUNCH_LOG_VIEW = 'LAUNCH_LOG_VIEW';
 export const SET_LOG_PAGE_LOADING = 'SET_LOG_PAGE_LOADING';
 export const FETCH_HISTORY_ITEMS_SUCCESS = 'FETCH_HISTORY_ITEMS_SUCCESS';
 export const UPDATE_HISTORY_ITEM_ISSUES = 'UPDATE_HISTORY_ITEM_ISSUES';
+export const LOG_PAGE_CHECKBOX_LABEL = 'History Across All Launches';
+export const FETCH_HISTORY_LINE = 'fetchHistoryLine';
+export const HISTORY_LINE_DEFAULT_VALUE = 'line';
+export const HISTORY_LINE_TABLE_MODE = 'table';
+export const HISTORY_LINE_LINE_MODE = 'line';
```

Comment:
You write the `line` value twice, like [here](https://github.com/reportportal/service-ui/pull/2597/files#diff-bbfeff900a85f6e21645cb4df2468b28cd04b5c2ceacae99018ada64bdac26f1R47). Please use it once.